bin/anthology/formatter.py

```diff
@@ -43,6 +45,35 @@
     "december": "dec",
 }
 
+_CSL_STYLES = {}
+
+
+def bibtype_to_csl(bibtype):
+    """Convert a BibTeX entry type to a CSL type.
+
```

Comment:
Can you expand the definition of CSL here, or [add a link](https://citationstyles.org/)? I'd never heard of it before.